### ISSUE TITLE
Parsing Arrays with XPath only return the first one

### ISSUE THREAD

**Issue by dzuluaga** [polotek/libxmljs#328]
I was expecting to return an array with all elements matching //VirtualHost XPath, so secure and default elements instead of the first one: 

```javascript
var libxmljs = require("libxmljs");
var xml =  '<?xml version="1.0" encoding="UTF-8" standalone="yes"?>' +
'<ProxyEndpoint name="default">' +
    '<HTTPProxyConnection>' +
        '<VirtualHost>secure</VirtualHost>' +
        '<VirtualHost>default</VirtualHost>' +
    '</HTTPProxyConnection>' +
'</ProxyEndpoint>';
var xmlDoc = libxmljs.parseXml(xml);
console.log(xmlDoc.get('//VirtualHost').toString());
```
However, it returns '<VirtualHost>secure</VirtualHost>'
Am I missing anything?

**Comment by hildjj** [comment]
Use `find` instead of `get`:

```console.log(xmlDoc.find('//VirtualHost/text()').toString());```

returns: `secure,default`

**Comment by dzuluaga** [comment]
Awesome! That worked. It'd be great to document this great feature or
include a reference from the tutorial. Thanks again!

On Thu, Aug 20, 2015 at 10:09 AM, Roman Shtylman <notifications@github.com>
wrote:

> Closed #328 <https://github.com/polotek/libxmljs/issues/328>.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/polotek/libxmljs/issues/328#event-387906360>.
>



-- 
Diego Zuluaga | apigee <http://www.apigee.com/> | m: +1.415.299.9983 | twitter
@dzuluaga <http://twitter.com/dzuluaga> @apigee <https://twitter.com/apigee>
| skype dzuluaga | *Apigee Community <http://community.apigee.com/>* for
answers, ideas and support!
[image: http://iloveapis.com/] <http://iloveapis.com/>
"First do it, then do it right, then do it better - this is my mantra for
successfully getting things done. It's all about the iteration." Addy Osmani
<https://twitter.com/addyosmani/status/314785735171518464>

**Comment by defunctzombie** [comment]
The wiki can be edited by anyone iirc.
